setup/MySQL/convert_mysql.sh

```diff
@@ -0,0 +1,107 @@
+#!/bin/bash
+
+#Set database name
+if [ $# -eq 2 ]; then
+   MYDATABASE="atpdatabase"
+else
+   MYDATABASE=$3
+fi
+
+#Get MySQL username and password from the arguments
+USERNAME=$1
+PASSWORD=$2
+
+#Get PWD
+PWD=`pwd`
+
+echo "Creating $MYDATABASE"
+
+mysql -u $USERNAME -p$PASSWORD << EOF
+
+#Create database from scratch
+create database $MYDATABASE;
+
+#Create players table
+CREATE TABLE $MYDATABASE.players (
+    id int,
+    firstname varchar(255),
+    lastname varchar(255),
+    hand varchar(255),
+    birth varchar(255),
```

Comment:
Can you add and populate an extra column called `birth_date` that stores the date in a more usable format? This is just a string, which is hard to query. Maybe we could use the MySQL Date datatype instead. Check out something similar done in lines [30](https://github.com/harman28/tennis_atp/blob/master/setup/PostgreSQL/convert_postgres.sh#L30), [94](https://github.com/harman28/tennis_atp/blob/master/setup/PostgreSQL/convert_postgres.sh#L94), and [114](https://github.com/harman28/tennis_atp/blob/master/setup/PostgreSQL/convert_postgres.sh#L114) of [convert_postgres.sh](https://github.com/harman28/tennis_atp/blob/master/setup/PostgreSQL/convert_postgres.sh) for reference.